components/MainNavBar.vue

```diff
@@ -61,7 +61,7 @@
              { name : 'Named Witches', url: '/', external: false },
              { name : 'Detention Locations', url: '/', external: false },
              { name : 'Trial Locations', url: '/', external: false },
-             { name : 'Trials in Each Modern Authority', url: '/', external: false },
+             { name : 'Trials in Each Modern Authority', url: '/council', external: false },
```

Comment:
I think the link should be changed to be `trials-in-each-modern-authority` to match the link text